index.Rmd

```diff
@@ -644,35 +582,62 @@ print(gdf_edges.info())
 
 <img src="./images/osmnxGraph.png" alt="Roads in Wageningen" width = "50%">
 
-OSMnx can store the downloaded street network (the Graph) as a shapefile or as a GeoDataFrame. Furthermore, the main purpose of the module is to perform network analyses, such as a shortest path from source to target location. Let us calculate the shortest path from Wageningen campus to Wageningen city center. Is this the route you would take?
+OSMnx can store the downloaded street network (the Graph) as a shapefile or as a `GeoDataFrame`. Furthermore, the main purpose of the module is to perform network analyses, such as a shortest path from source to target location. Let us calculate the shortest path from Wageningen campus to Wageningen city center. Is this the route you would take?
 
 ```{Python, eval=FALSE}
 # Origin
-source = ox.distance.get_nearest_node(wageningenRoadsGraph, (51.987817, 5.665779))
+source = ox.distance.nearest_nodes(wageningenRoadsGraph, 5.665779, 51.987817)
+
 # Destination
-target = ox.distance.get_nearest_node(wageningenRoadsGraph, (51.964870, 5.662409))
+target = ox.distance.nearest_nodes(wageningenRoadsGraph, 5.662409, 51.964870)
+
 # Compute shortest path 
 shortestroute = ox.distance.shortest_path(G=wageningenRoadsGraph,
-                                    orig=source, dest=target, weight='length')
+                                          orig=source, 
+                                          dest=target, 
+                                          weight='length')
+
 # Plot
-ox.plot.plot_graph_route(wageningenRoadsGraph, shortestroute, figsize=(20,20),
-                         route_alpha=0.6, route_color='darkred',
+ox.plot.plot_graph_route(wageningenRoadsGraph, shortestroute, figsize=(20, 20), 
+                         route_alpha=.6, route_color='darkred',  bgcolor='#FFFFFF', 
+                         node_color='#AAAAAA', edge_color='#AAAAAA',
```

Comment:
Can names be used for the colours? Otherwise the students might be thinking: "AAAAAA! I don't know what that means!!!" :D